05/SendEmail.js

```diff
@@ -0,0 +1,18 @@
+import emailjs from '@emailjs/browser';
+import acc from './account';
+
+const SendEmail = (form) => {
```

Comment:
Ani to komponent, ani konstruktor wiec nazwę funkcji napisałbym z małej litery :)